smartapps/smartthings/logitech-harmony-connect.src/logitech-harmony-connect.groovy

```diff
@@ -658,29 +658,75 @@ def updateDevice() {
 	def data = request.JSON
 	def command = data.command
 	def arguments = data.arguments
-
+  def type = params.deviceType
 	log.debug "updateDevice, params: ${params}, request: ${data}"
 	if (!command) {
 		render status: 400, data: '{"msg": "command is required"}'
 	} else {
 		def device = allDevices.find { it.id == params.id }
-		if (device) {
-        	if (device.hasCommand("$command")) {
+    if (device) {
+        if (validateCommand(device, type, command)) {
                 if (arguments) {
                     device."$command"(*arguments)
                 } else {
                     device."$command"()
                 }
                 render status: 204, data: "{}"
```

Comment:
I was talking about the `render status: 204, data: "{}"` line here. Do we need this? Because we are simple executing a command on the device. I may be missing something.